service/resource/vaultcrt/create.go

```diff
@@ -77,48 +75,3 @@ func (r *Resource) newCreateChange(ctx context.Context, obj, currentState, desir
 
 	return secretToCreate, nil
 }
-
-func (r *Resource) ensureVaultRole(customObject certificatetpr.CustomObject) error {
```

Comment:
This is now used for creation and deletion, so I moved it to the `resource.go` to have the code in a more common place.